src/index.js

```diff
@@ -9,7 +9,40 @@ const {
 } = require('./cron/schedule-google-calendar');
 const { firebaseConfig } = require('../firebase-config');
 const cron = require('cron');
+const saveErrorLog = require('./utils/log-error');
 
+async function startClientBot(client) {
+  client.commands = new Collection();
+  deployCommands(client);
+  deployEvents(client);
+
+  scheduleMessages(client, SCHEDULE_MESSAGES.messageTimes);
+  if (firebaseConfig.scheduledCalendarEnabled) {
+    new cron.CronJob(
+      SCHEDULE_MESSAGES.scheduledCalendarInterval,
+      () => {
+        console.log('Running scheduled calendar notifications...');
+        scheduleCalendarNotifications(client);
+      },
+      null,
+      true,
+      SCHEDULE_MESSAGES.timeZone
+    );
+
+    console.log(
+      'Calendar event collector scheduled to run from Monday to Friday, 8 AM to 5 PM each 20 MIN (Colombia time).'
```

Comment:
I think this is a dynamic time but the console log doesn't handle dynamic time, maybe you can put here the cronjob configuration (it looks something like `* * * * 1-5 0 8 0 0`), it is not so friendly but it would be dynamic

Reply:
Which one is that config that you mentioned here?